### PR TITLE
chore(CCOM-420): updates catho libs

### PR DESCRIPTION
**Pull request by toncabral88**
[ ] Code review
[ ] Build and start

### CODE REVIEW

**Review by MarcosViniciusPC — CHANGES_REQUESTED** [review]
Captura de tela de 2022-08-09 14-35-42

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "next": "^11.1.2",
     "next-fonts": "^0.17.0",
     "next-images": "^1.1.2",
-    "node-sass": "^4.13.1",
+    "node-sass": "^6.0.0",
```
Comment (MarcosViniciusPC) [inline package.json:23]: What is this package used for?

**Review by MarcosViniciusPC — APPROVED** [review]
[x ] Code review
[x] Build and start

**Review by rapahaeru — APPROVED** [review]
[x] Code review
[x] Build and start

**Pull request merged**